saskatoon/sitebase/templates/app/detail_views/harvest/breadcomb.html

```diff
@@ -13,7 +14,7 @@
                                 </div>
                                 <div class="breadcomb-ctn">
                                     <h1>#{{ harvest.id }}:
-                                        {% for t in harvest.trees %} {{ t }}, {% endfor %}
+                                        {% for tree in trees %} {{ tree.name }} {{ tree.fruit_name|get_fruit_name_icon }}, {% endfor %}
```

Comment:
btw @mina-andajos  Are you sure you are using your serializer and not the trees objects I just mentioned in my comment?

Reply:
In `breadcomb.html` & `about.html`. It's using the tree objects. didn't change them.
I used the serializer for the `HarvestViewset.list()` used in :

- saskatoon/sitebase/templates/app/list_views/harvest/data_row.html
- saskatoon/sitebase/templates/app/list_views/property/data_row.html